walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -91,42 +92,43 @@
 
   /**
    * Recursive helper function used to find all combinations of size r of the input array arr[].
-   * chosen[]: temporary array to store current combination.
-   * start & end: staring and ending indexes in arr[].
-   * index: current index in data[].
+   * @param arr[] The input array.
+   * @param chosen[] Temporary array to store current combination.
+   * @param start Starting index in arr[].
+   * @param end Ending index in arr[].
+   * @param index Current index in chosen[].
+   * @param r The size of combinations to be found.
    */
   private void combinationUtil(String arr[], String chosen[], int start, int end, int index, int r) { 
```

Comment:
This line is longer than 80 chars.

Reply:
Splitting it. How strict are we about the 80 chars things?